Source/WebCore/platform/graphics/GraphicsLayer.h

```diff
@@ -797,6 +800,7 @@ class GraphicsLayer : public RefCounted<GraphicsLayer> {
     bool m_isDescendentOfSeparatedPortal : 1;
 #endif
 #endif
+    bool m_headlessModeEnabled : 1;
```

Comment:
This should not be state on every GraphicsLayer. Instead, extend the `GraphicsLayerClient` protocol.